### PR TITLE
Minor typo fixes

### PR DESCRIPTION
**Pull request by eemeli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,13 +53,13 @@ Here are the steps to have your company sign the CLA.
 1. Open a PR against a repo that uses EasyCLA.
 1. When blocked by the bot, follow the prompts and choose *Corporate*.
 1. Choose your company from the list. If it's not there, add it.
-1. Designate someone with signing authority (generally an officer or attorney, if in doubt ask your manager) to receive the DocuSign. This person is the *CLA Manager*.
+1. Designate someone with signing authority (generally an officer or attorney, if in doubt ask your manager) to receive the DocuSign form. This person is the *CLA Manager*.
```
Comment (ljharb) [inline README.md:56]: the change is correct; but the original is correct too - “the DocuSign” is colloquially used as the noun.

**Review by ljharb — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by eemeli** [comment]
Just realised that it might not have been clear, but this was my "trivial PR against this repo" that the README instructs for maintainers to do. So given that I've managed to subsequently jump through the appropriate hoops with the CLA bot, I have no problems with this getting merged or rejected, as determined by anyone who feels empowered to do so.

**Comment by brianwarner** [comment]
Thanks, yeah, just to be clear, this text is written so people can nitpick it (and give them a reason to open a PR).

**Pull request closed without merging**